search/django/weedid/views.py

```diff
@@ -56,6 +57,27 @@ def upload_image(request):
         return HttpResponse("Only support POST request")
 
 
+def upload_agcontexts(request):
```

Comment:
I had imagined that posting the agcontext would just store it in a file, and then before Deposit there would be a conversion step that integrates the additional content. This might generalise better to other upload formats. For now, this solution is fine,.